lib/rubocop/cop/lint/number_conversion.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop check usage of unsafe number conversions, eg. `foo.to_i`.
+      #
+      # @example
+      #
+      #   # bad
+      #   `foo.to_i`
```

Comment:
Don't use back-ticks in the code examples - this is going to look bad in the manual.

Reply:
Removed all back-ticks.